src/lib/pages/backoffice/EItem/EItemEditor/schema.js

```diff
@@ -74,7 +84,7 @@ export const schema = () => {
         type: 'array',
       },
     },
-    required: ['document_pid'],
+    required: ['document_pid', 'eitem_type'],
```

Comment:
I don't remember if possible to set a default value for required field in the form: in most of the cases it will be an e-book, so we could help the librarians to have it pre-filled